src/server/middleware/clientErrorLogger.js

```diff
@@ -42,11 +41,11 @@ export default function clientErrorLogger(req, res) {
           return;
         }
         const {
-          msg, stack, href, otherData,
+          msg, stack, type, href, otherData,
         } = raw;
         const err = new Error(msg);
         Object.assign(err, {
-          name: 'ClientReportedError',
+          name: type,
```

Comment:
as this is the clientErrorLogger, what is the scenario where its not a client reported error ?

Reply:
I can revert to hardcoding this value, however I need to pass a new value `ServerSideReportedError` here https://github.com/americanexpress/one-app-ducks/pull/32/files#diff-5dc992175b262921c364a57cee1fcd0f6e81cd4d086d8115a20238ca7e14b119R40 so we can adjust the schema accordingly in the production logger here: https://github.com/americanexpress/one-app/pull/508/files#diff-481c987f1307f2ec947cbca5fd31e173890266a92b6148ba3f177e90d8136cc0

Reply:
Sorry, I'm not sure I follow 😓 : what's the need behind overriding the Error instance's name property with `ServerSideReportedError`?

Reply:
Reason was to identify the difference between the `clientReportedError` and the server error so we could append the metadata, however, your suggestion of doing it by checking for the metaData property, however, for the client reported error the metaData will appear twice.

`if (err.metaData) {`

Inside the `err.request.metaData` and `err.metaData`

Reply:
We don't have to use the `name` property since you mention it could cause confusion and debugging issues with the prototype. What about using another property to identify if it is an error reported by the client or the server? like `errorType`?